src/main/java/echoquery/sql/StarSchemaQuerier.java

```diff
@@ -0,0 +1,60 @@
+package echoquery.sql;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.DriverManager;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import echoquery.utils.EchoQueryCredentials;
+
+
+public class StarSchemaQuerier {
+  private static final StarSchemaQuerier querier = new StarSchemaQuerier();
+  private static final Logger log =
+      LoggerFactory.getLogger(StarSchemaQuerier.class);
+  private final Connection conn = this.getConnection();
```

Comment:
Confused how this actually even works, "this" is referenced before the constructor is even called?